ExploreASL_Initialize.m

```diff
@@ -509,23 +462,10 @@ function xASL_init_AddDirsOfxASL(MyPath)
     
 end
 
-
```

Comment:
I added these spaces for a reason I believe, that there is a lot of space between main code sections/subfunctions and less space between different code sections (not complaining on your edit, just explaining why I did this)

Reply:
Yesyes. I know - but then we added the double %%, which actually works now nicely, so I thought we can remove the space. If you agree?